WordPress/src/main/java/org/wordpress/android/ui/FilteredRecyclerView.java

```diff
@@ -133,27 +135,27 @@ private void init() {
         mProgressLoadMore = (ProgressBar) findViewById(R.id.progress_loading);
         mProgressLoadMore.setVisibility(View.GONE);
 
-        mSwipeToRefreshHelper = new SwipeToRefreshHelper(getContext(),
-                (CustomSwipeRefreshLayout) findViewById(R.id.ptr_layout),
-                new SwipeToRefreshHelper.RefreshListener() {
-                    @Override
-                    public void onRefreshStarted() {
-                        post(new Runnable() {
-                            @Override
-                            public void run() {
-                                if (!NetworkUtils.checkConnection(getContext())) {
-                                    mSwipeToRefreshHelper.setRefreshing(false);
-                                    updateEmptyView(EmptyViewMessageType.NETWORK_ERROR);
-                                    return;
-                                }
-                                if (mFilterListener != null){
-                                    mFilterListener.onLoadData();
-                                }
+        mSwipeToRefreshHelper = buildSwipeToRefreshHelper(
+            (CustomSwipeRefreshLayout) findViewById(R.id.ptr_layout),
```

Comment:
This is of course rather minor but, it seems the continuation indentation was reduced to 4 spaces instead of 8 that we usually have (even in this same file anyway). Let's revert it back to 8 spaces (in all files that got to down to 4) which will help reduce the diff size of the PR as well. WDYT @theck13 ?

Reply:
The indention was updated in 64cd53ff27ec8a0394e68fe64535719e1437a2d1.